profiles/common/modules/custom/ecas/ecas.api.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+/**
+ * @file
+ * Documentation file for the module API.
+ */
+
+/**
+ * Act on the list of user data value to save of the Ecas user.
+ *
+ * @param array $edit
+ *   The list array of user data values. it is the same "$edit" parameter as
+ *   in user_save().
+ * @param object $user
+ *   A user object.
+ * @param array $user_info
+ *   An associative array with the following interesting keys:
+ *   - mail: mail address.
+ *   - givenname: first name.
+ *   - sn: last name.
+ *   These values will be used to fill fields/profiles/...
+ * @param array $args
+ *   Extra parameters, not used directly in this function but passed to the
+ *   info_ecas_update() hook.
+ */
+function hook_ecas_sync_user_info(array &$edit, $user, array $user_info, array $args) {
```

Comment:
Should we not keep the same argument ordering through functions? ($user, $user_info, $edit, ..... as we see in functions like ecas_sync_profile_core and even in user_save)